charts/eol-exporter/values.yaml

```diff
@@ -0,0 +1,230 @@
+      ENVIRONMENT:
+        value: prod
+      CONFIG_YAML_PATH:
+        value: config.yaml
+      JOB_INTERVAL_HOURS:
+        value: '24'
+    ports:
+      - name: http
+        containerPort: 8080
+        protocol: TCP
+    volumeMounts:
+      config:
+        mountPath: /workspace/app/config.yaml
+        subPath: config.yaml
+    volumes:
+      config:
+        configMap:
+          name: eol-exporter-config
+    startupProbe:
+      enabled: false
+      failureThreshold: 30
+      periodSeconds: 10
+      # Must specify either one of the following field when enabled
+      httpGet: {}
+      exec: {}
+      tcpSocket: {}
+    readinessProbe:
+      enabled: true
+      failureThreshold: 3
+      periodSeconds: 10
+      successThreshold: 1
+      timeoutSeconds: 1
+      initialDelaySeconds: 10
+      # Must specify either one of the following field when enabled
+      httpGet:
+        # Prometheus client WSGI server has a /favicon.ico path that gives a 200 OK response that can
+        # be configured as health check endpoint. See:
+        # https://github.com/prometheus/client_python/blob/master/prometheus_client/exposition.py#L121
+        path: /favicon.ico
+        port: 8080
+      exec: {}
+      tcpSocket: {}
+    livenessProbe:
+      enabled: true
+      failureThreshold: 3
+      periodSeconds: 10
+      successThreshold: 1
+      timeoutSeconds: 1
+      initialDelaySeconds: 10
+      # Must specify either one of the following field when enabled
+      httpGet:
+        path: /favicon.ico
+        port: 8080
+      exec: {}
+      tcpSocket: {}
+    resources:
+      limits:
+        memory: 256Mi
+        cpu: 500m
+      requests:
+        memory: 128Mi
+        cpu: 100m
+    securityContext:
+      runAsNonRoot: true
+      runAsUser: 1002
+      runAsGroup: 1000
+      # fsGroup: 2000
+      # fsGroupChangePolicy: "OnRootMismatch"
+    containerSecurityContext:
+      allowPrivilegeEscalation: false
+      privileged: false
+      runAsNonRoot: true
+      runAsUser: 1002
+      runAsGroup: 1000
+      readOnlyRootFilesystem: false
+      # capabilities:
+      #   drop:
+      #     - ALL
+      #   add:
+      #     - CHOWN
+      #     - NET_ADMIN
+      #     - SYS_TIME
+    strategy:
+      type: RollingUpdate
+      rollingUpdate:
+        maxSurge: 25%
+        maxUnavailable: 25%
+    additionalLabels: {}
+    tolerations: []
+    nodeSelector: {}
+    affinity:
+      nodeAffinity:
+        requiredDuringSchedulingIgnoredDuringExecution:
+          nodeSelectorTerms:
+            - matchExpressions:
+                - key: kubernetes.io/arch
+                  operator: In
+                  values:
+                    - amd64
+    topologySpreadConstraints: {}
+    revisionHistoryLimit: 5
+  additionalContainers: []
+  service:
+    enabled: true
+    annotations: {}
+    ports:
+      - port: 8080
+        name: http
+        protocol: TCP
+        targetPort: 8080
+    type: ClusterIP
+  ingress:
```

Comment:
this solution will not have an ingress - so don't have it declared

Reply:
Removed